javascriptv3/example_code/lambda/scenarios/basic/README.md

```diff
@@ -0,0 +1,34 @@
+# Scenario - AWS Lambda for AWS SDK for JavaScript (v3)
+
+This is an interactive command prompt that showcases how the JavaScript SDK can be
+used for interacting with Lambda.
+
+For the purpose of this example `root` refers to the same directory as this readme.
+
+## Prerequisites
+
+- create an AWS account
+- setup a [credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
+- configure your account (or IAM user) with access to Lambda and IAM
+- install `nodejs` version 18.7.0
+- install `yarn`
```

Comment:
- Create an AWS account.
- Set up a [credentials file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
- Configure your account (or AWS Identity and Access Management (IAM) user) with access to Lambda and IAM.
- Install `nodejs` version 18.7.0.
- Install `yarn`.